tests/commons/src/main/java/org/codice/ddf/admin/comp/graphql/GraphQlHelper.java

```diff
@@ -90,6 +114,13 @@ private String getResourceAsString(String filePath) {
     return null;
   }
 
+  private GraphQLRequest populateRequestArgs(GraphQLRequest request, Map<String, Object> args) {
```

Comment:
Can't we add this as a method to the `GraphQLRequest` called `addArguments` instead?